### ISSUE TITLE
.track() returns null

### ISSUE THREAD

**Issue by chadsaun** [svgdotjs/svg.js#824]
I'm having an issue with .track() returning null for the path. Here I'm creating the text along a path which works just fine.

```
let _text = this.draw.text((add) => {
  add.tspan(text.text)
})
_text.font(fontConfig)
_text.path(textPath)
let _path = _text.track()
```

The only difference when it **doesn't work**, is after I've inserted an SVG into the main SVG using `.svg()`. I've traced the issue down to where `.track()` calls `path.reference('href')`...

```
// Get the path track element
  , track: function() {
      var path = this.textPath()

      if (path)
        return path.reference('href')
    }
```

...which calls `SVG.get()` and here `document.getElementById()` can't find the ID (even though the ID it's looking for is correct).

```
// Method for getting an element by id
SVG.get = function(id) {
  var node = document.getElementById(idFromReference(id) || id)
  return SVG.adopt(node)
}
```

I'm wondering if the SVG hasn't been added to the document yet? Are there any cases where this would happen?

**Comment by Fuzzyma** [comment]
Describing from what you say the only possibility this dies not work the ID is wrong or maybe not unique. Can you post the code with the import with svg()?

**Comment by chadsaun** [comment]
Yah here it is...

```
// Grabs the SVG source
let graphicSvg = sessionStorage.getItem(graphic.image)

// Assign a new ID to the SVG to make sure it's unique
let graphicId = `graphic${index1}${index2}`
let rgx = '/<svg[^>]*id="([a-z0-9_-]+)"[^>]*>/gi'
let matches = graphicSvg.match(rgx)
if (matches) {
  let existingId = matches[1]
  graphicSvg = graphicSvg.replace(existingId, graphicId)
} else {
  graphicSvg = graphicSvg.replace('<svg', `<svg id="${graphicId}"`)
}
this.draw.svg(graphicSvg).addClass(pathClassName)
prefixGraphic = SVG.get(graphicId)
```

**Comment by Fuzzyma** [comment]
Didnt you say `track()` is not working? Cant see it in that snipped. Or is the last SVG.get already not working?

**Comment by chadsaun** [comment]
Ok I tracked down my issue. It was with this line here. 
```
this.draw.svg(graphicSvg).addClass(pathClassName)
```
It was adding a class name to the whole SVG when I wanted it to add it to the SVG I just inserted. 

I replace the text (which is on a path) every time the text changes, but SVG.js leaves the referenced path in the `<defs>` after I remove the textpath. So I have code that cleans those old ones up by using the class name.
